pyro/params/param_store.py

```diff
@@ -170,15 +170,20 @@ def save(self, filename):
         with open(filename, "wb") as output_file:
             torch.save(self.get_state(), output_file)
 
-    def load(self, filename):
+    def load(self, filename, load_to_cpu=False):
```

Comment:
Why not simply thread through the `map_location` arg to `torch.load`? Then you could achieve what you want with
```py
pyro.load(filename, map_location=lambda s,l: 'cpu')
```
and also load to gpu via e.g.
```py
pyro.load(filename, map_location=lambda s,l: 'cuda:0')
```